lib/llm/src/discovery/watcher.rs

```diff
@@ -178,7 +181,12 @@ impl ModelWatcher {
 
     /// If the last instance running this model has gone delete it.
     /// Returns the name of the model we just deleted, if any.
-    async fn handle_delete(&self, kv: &KeyValue) -> anyhow::Result<Option<String>> {
+    async fn handle_delete(
+        &self,
+        kv: &KeyValue,
+        target_namespace: Option<&str>,
+        global_namespace: bool,
```

Comment:
nit, but can you rename this to `is_global_namespace`, to make it clear it's a boolean, and differentiate it from `target_namespace` which is a string.